src/index.ts

```
@@ -277,14 +295,28 @@ export function isShieldActiveWithSelection(
   );
 }
 
+export function blockAppsWithSelectionId(
+  familyActivitySelectionId: string,
+  triggeredBy?: string,
+): void {
+  return ReactNativeDeviceActivityModule?.blockAppsWithSelectionId(
+    familyActivitySelectionId,
+    triggeredBy,
+  );
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)